### DISCUSSION TITLE
Error in RTSP docker install - "invalid reference format: repository name must be lowercase"

### DISCUSSION THREAD

**Discussion by itm1960** [fuatakgun/eufy_security#729] [Q&A]
I get this error when I try to set up the RTSP server using the command in the instructions:
`docker run -it RTSP_PROTOCOLS=tcp -p 8554:8554 -p 1935:1935 aler9/rtsp-simple-server:latest`

Any idea what the correct command should be?

**Comment by murf1** [comment]
Hi,
looks like the -e is missing. Should be:

docker run -it -e RTSP_PROTOCOLS=tcp -p 8554:8554 -p 1935:1935 aler9/rtsp-simple-server:latest
